.github/workflows/build.yml

```diff
@@ -16,6 +16,9 @@ on:
 
 env:
   MAIN_CRATE: portable-rustls
+  # NOTE: USING this undocumented cfg option to simplify running some of the existing CI jobs
+  # TODO: consider a better solution - it would be nice to avoid breaking too many of the existing CI jobs, if possible
+  RUSTFLAGS: --cfg unsupported_std_arc_cfg
```

Comment:
I think it would be ideal to try testing some of the build jobs in this workflow with default build (no cfg option).